privacy.md

```diff
@@ -0,0 +1,76 @@
+---
+layout: page
+page: privacy
+title: Privacy Policy
+nav: main
+city_page: false
+---
+{::options parse_block_html="true" /}
+<div class="page-block">
+{::options parse_block_html="true" /}
+<div class="unihack-wrap container-fluid">
+{::options parse_block_html="true" /}
+<div class="row justify-content-md-center">
+{::options parse_block_html="true" /}
+<div class="col-lg-8 col-md-12 col-xs-12">
+
+**UNIHACK INCORPORATED** is the not-for-profit organisation committed to running enjoyable, informative and inclusive hackathons for university students. We are bound by this policy which outlines our ongoing obligations to you in respect of how we manage your Personal Information.
+
+We have adopted the Australian Privacy Principles (APPs) contained in the Privacy Act 1988 (Cth) (the Privacy Act). The APPs govern the way in which we collect, use, disclose, store, secure and dispose of your Personal Information.
+
+A copy of the Australian Privacy Principles may be obtained from the website of The Office of the Australian Information Commissioner at <https://oaic.gov.au>.
+
+### What is Personal Information and why do we collect it?
+
+Personal Information is information or an opinion that identifies an individual. Examples of Personal Information we collect include: names, gender, addresses, email addresses, phone numbers, tertiary institutions, education information, and dietary requirements and/or medical conditions that you disclose to us in order to assist us help you throughout the event.
+
+This Personal Information is obtained in several ways including forms on our website <{{ site.url }}>, correspondence via email or phone calls, and the third parties listed below. We don’t guarantee website links or policy of authorised third parties.
+
+We collect your Personal Information for the primary purpose of providing our services to you and providing information to sponsors. You may unsubscribe from our mailing/marketing lists and opt-out from us providing information to our sponsors at any time by contacting us in writing.
+
+When we collect Personal Information we will, where appropriate and where possible, explain to you why we are collecting the information and how we plan to use it.
+
+### Third Parties
+
+We will primarily collect your Personal Information only from you. However, we also will be provided with information by several third parties, such as our third-part ticketing platform [Eventbrite](https://www.eventbrite.com.au/), payment provider [Pin Payments](https://pinpayments.com/), communications system [Slack](https://slack.com/) and hackathon management service, [DevPost](https://devpost.com/). In such a case we will take reasonable steps to ensure that you are made aware of the information provided to us by the third party.
```

Comment:
However, we also will be provided with information by several third**y** parties, such as our third-part ticketing platform ?

Reply:
Also extra comma `hackathon management service~~,~~ [DevPost](https://devpost.com/)`

Reply:
Speaking of C&S, we should probs list the host institution as a third party - nay ? yay ?

Reply:
Do you know what data we could be providing to/receiving from a host @sph-huynh?

Like potentially us providing them with a list of names for security purposes, or something along those likes?